curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md

```diff
@@ -6,32 +6,38 @@ challengeType: 2
 Schemas are building block for Models. They can be nested to create complex models, but in this case we’ll keep things simple.
 A model allows you to create instances of your objects, called documents.
-Create a person having this prototype :
-<code>- Person Prototype -</code>
-<code>--------------------</code>
-<code>name : string [required]</code>
-<code>age :  number</code>
-<code>favoriteFoods : array of strings (*) </code>
-Use the mongoose basic schema types. If you want you can also add
-more fields, use simple validators like required or unique,
-and set default values. See the <a href='http://mongoosejs.com/docs/guide.html'>mongoose docs</a>.
-[C]RUD Part I - CREATE
-Note: Glitch is a real server, and in real servers the interactions with the db happen in handler functions. These function are executed when some event happens (e.g. someone hits an endpoint on your API). We’ll follow the same approach in these exercises. The done() function is a callback that tells us that we can proceed after completing an asynchronous operation such as inserting, searching, updating or deleting. It’s following the Node convention and should be called as done(null, data) on success, or done(err) on error.
-Warning - When interacting with remote services, errors may occur !
-<code>/* Example */</code>
-<code>var someFunc = function(done) {</code>
-<code>&nbsp;&nbsp;//... do something (risky) ...</code>
-<code>&nbsp;&nbsp;if(error) return done(error);</code>
-<code>&nbsp;&nbsp;done(null, result);</code>
-<code>};</code>
+
+<strong>Note:</strong> Glitch is a real server, and in real servers the interactions with the db happen in handler functions. These function are executed when some event happens (e.g. someone hits an endpoint on your API). We’ll follow the same approach in these exercises. The <code>done()</code> function is a callback that tells us that we can proceed after completing an asynchronous operation such as inserting, searching, updating or deleting. It’s following the Node convention and should be called as <code>done(null, data)</code> on success, or <code>done(err)</code> on error.
+Warning - When interacting with remote services, errors may occur!
+<blockquote>
+/* Example */<br><br>
+var someFunc = function(done) {<br>
+&nbsp;&nbsp;//... do something (risky) ...<br>
+&nbsp;&nbsp;if(error) return done(error);<br>
+&nbsp;&nbsp;done(null, result);<br>
+};
+</blockquote>
 </section>
 
 ## Instructions
 <section id='instructions'>
+Create a person having this prototype :
+<blockquote>
+- Person Prototype -<br>
+--------------------<br>
+name : string [required]<br>
+age :  number<br>
+favoriteFoods : array of strings (*)
+</blockquote>
 
+Use the mongoose basic schema types. If you want you can also add
+more fields, use simple validators like required or unique,
+and set default values. See the <a href='http://mongoosejs.com/docs/guide.html'>mongoose docs</a>.
```

Comment:
What about moving it to the guide article for this challenge?

Reply:
That is what I meant. Silly me, I meant IMO we should move this to the guide

Reply:
The current Guide article for this is a stub, so for now, I am just going to leave it here and when this challenge is rewritten, it can be moved then.

Reply:
Alright, that sounds great too!